.env.example

```
@@ -36,6 +36,13 @@ VECTOR_STORE_TYPE=chromadb
 LOG_LEVEL=INFO  # DEBUG, INFO, WARNING, ERROR, CRITICAL
 LOG_FILE=logs/aitext.log
 
+# ── Token Watcher Configuration ──
+# Enable token usage monitoring (1=enabled, 0=disabled)
+TOKEN_WATCHER=1
+# Number of logs per page in the frontend
+TOKEN_PAGINATE=10
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TOKEN_PAGINATE key should go before the TOKEN_WATCHER key

(UnorderedKey)